content/documentation/pages/6-recipes/2-file-ingest/1-sftp-to-jdbc.md

```diff
@@ -0,0 +1,936 @@
+
+### SFTP server
+
+This example requires access to an SFTP server. For running on a `local` machine and `minikube`, we will use the host machine as the SFTP server. For `Cloud Foundry`, and `Kubernetes` in general, an external SFTP server is required.
+On the SFTP server, create a `/remote-files` directory. This is where we will drop files to trigger the pipeline.
+
+### NFS configuration
+
+<!--TIP-->
+
+NFS is not required when running locally.
+
+<!--END_TIP-->
+
+#### Cloud Foundry NFS configuration
+
+This feature is provided in Pivotal Cloud Foundry by [NFS Volume Services](https://docs.pivotal.io/pivotalcf/2-5/devguide/services/using-vol-services.html)
+
+To run this example, we will need:
+
+- a Cloud Foundry instance v2.3+ with NFS Volume Services [enabled](https://docs.pivotal.io/pivotalcf/2-5/opsguide/enable-vol-services.html)
+- An NFS server accessible from the Cloud Foundry instance
+- An `nfs` service instance properly configured
+
+<!--NOTE-->
+
+**NOTE:** For simplicity, this example assumes the `nfs` service is created with common configuration as follows with a common mount point `/var/scdf` for all bound apps. It is also possible to set these parameters when binding the nfs service to an application using [deployment propterties](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configure-service-binding-parameters):
+
+```bash
+$cf create-service nfs Existing nfs -c '{"share":<nfs-host>/staging","uid":<uid>,"gid":<gid>, "mount":"/var/scdf"}'
+```
+
+<!--END_NOTE-->
+
+#### Kubernetes NFS configuration
+
+Kubernetes provides many options for configuring and sharing persistent volumes. For this example, we will use `minikube` and use the host machine as the NFS server. The following instructions works for `OS/X` and should be similar for Linux hosts:
+
+Make sure minikube is started. The commands below provide NFS access to the minikube VM. The minikube IP is subject to change each time it is started, so these steps should be performed after each start.
+
+Here we will expose a shared directory called `/staging`.
+
+```bash
+$ sudo mkdir /staging
+$ sudo chmod 777 /staging
+$ sudo echo "/staging -alldirs -mapall="$(id -u)":"$(id -g)" $(minikube ip)" >> /etc/exports
+$ sudo nfsd restart
+```
+
+Verify the nfs mounts:
+
+```bash
+$ showmount -e 127.0.0.1
+Exports list on 127.0.0.1:
+/staging 192.168.99.105
+```
+
+Configure persistent volume and persistent volume claim resources. Copy the following to `nfs-config.yml`:
```

Comment:
If `nfs-config.yml` is a hosted file, it'd be good to link to it also.

Reply:
Asking the user to create the file.